www/templates/default/html/manager/EventFormImageUpload.tpl.php

```diff
@@ -0,0 +1,70 @@
+<fieldset>
+	<legend class="dcf-legend dcf-txt-md">Image</legend>
+	<div class="details">
+		<?php if ($context->event->imagemime != NULL): ?>
+            <div class="dcf-mb-4">
+                <img src="<?php echo $base_frontend_url ?>images/<?php echo $context->event->id; ?>" alt="image for event <?php echo $context->event->id; ?>">
+                <?php if(!$context->on_main_calendar): ?>
+                    <br>
+                    <input type="checkbox" name="remove_image" id="remove-image">
+                    <label for="remove-image" class="dcf-txt-sm">Remove Image</label>
+                <?php endif; ?>
+            </div>
+		<?php endif; ?>
+        <label class="dcf-label" for="imagedata"><span class="required-for-main-calendar dcf-required" style="display: none">* </span>Image Upload</label>
+		<input class="dcf-mt-4 dcf-input-file" style="font-size: 10px;" type="file" name="imagedata" id="imagedata" accept="image/gif, image/jpeg, image/png, image/avif, image/webp" aria-label="Select an Image">
+        <div class="dcf-mt-4 dcf-form-help">
+            <p>Do not include text on image, any text should be included in title and description of event. Image should be descriptive of event.</p>
```

Comment:
Split first sentence into two: "Do not include text on image. Any text should be included in title and description of event."